Configurations/descrip.mms.tmpl

```diff
@@ -553,6 +553,9 @@ configdata.pm : $(SRCDIR)Configure $(SRCDIR)config.com {- join(" ", @{$config{bu
         @ WRITE SYS$OUTPUT "*************************************************"
         @ PIPE ( EXIT %X10000000 )
 
+reconfigure reconf:
```

Comment:
I am not thrilled with two targets that are exactly the same, especially since this is a brand-new thing (not for backward compatibility).  but you already merged, so oh well.

Reply:
Well, they aren't *exactly* exactly the same.  One works with dependencies and is used internally in a number of places (`opensslconf.h` depends on `configdata.pm`, for example, just like anything else that's generated from `configdata.pm` data), while the other is there purely for the users convenience and is intended to be used manually.

Sure, I could have made one do `$(MAKE) reconfigure`, but I dunno, I have a profound dislike for recursive make invocations if it can be avoided.